waf.go

```diff
@@ -95,3 +106,47 @@ func (api *API) ListWAFRules(zoneID, packageID string) ([]WAFRule, error) {
 	}
 	return rules, nil
 }
+
+// WAFRule returns a WAF rule from the given WAF package.
+//
+// API Reference: https://api.cloudflare.com/#waf-rules-rule-details
+func (api *API) WAFRule(zoneID, packageID, ruleID string) (WAFRule, error) {
+	var r WAFRuleResponse
```

Comment:
I tend to define the variables as close as possible to the use of them to make things easier for reviewing but that's just a personal nitpick you can ignore if you'd like.